src/pybind/mgr/orchestrator.py

```diff
@@ -1219,8 +1219,8 @@ def validate_add(self):
 
 
 class NFSServiceSpec(StatelessServiceSpec):
-    def __init__(self, name, pool=None, namespace=None, count=1, placement=None):
-        super(NFSServiceSpec, self).__init__(name, placement, count)
```

Comment:
This will break `orchestrator_cli.module.OrchestratorCli#_nfs_update`, but `orchestrator_cli.module.OrchestratorCli#_nfs_add` is broken already, as you cannot create a `NFSServiceSpec` at all at the moment.